pipelines/A10/queries.log

```diff
@@ -1,30 +1,45 @@
+[1] "Environment: REMOTE, Time: 2023-10-10 07:09:30.450475"
 
 Query Nr. 1
--- In welchem Jahr gab es die meisten Straftaten in der Schweiz?
-SELECT T.jahr as jahr_mit_meisten_straftaten FROM straftaten_aufklaerung as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de = 'Schweiz' AND S.country = TRUE AND T.straftat = "Straftat - Total" GROUP BY jahr ORDER BY anzahl_straftaten DESC LIMIT 1;
- jahr_mit_meisten_straftaten
-                        2012
+-- In which year were there the most crimes in Switzerland? Also please provide the number of committed crimes.
+SELECT T.year AS year_with_most_crimes, T.number_criminal_offences_registered FROM criminal_offences_registered_by_police AS T JOIN spatial_unit AS S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name='Switzerland'   AND S.country = TRUE   AND T.offence_criminal_code='Offence - total' GROUP BY T.year, T.number_criminal_offences_registered ORDER BY T.number_criminal_offences_registered DESC LIMIT 1;
+ year_with_most_crimes number_criminal_offences_registered
+                  2012                              611903
 
 Query Nr. 2
--- Welches ist die meist begangene Straftat in der Schweiz?
-SELECT T.straftat as hauefigste_straftat FROM straftaten_aufklaerung as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de = 'Schweiz' AND S.country = TRUE AND T.straftat != "Straftat - Total" GROUP BY straftat ORDER BY anzahl_straftaten DESC LIMIT 1;
-          hauefigste_straftat
- Fahrzeugdiebstahl (Art. 139)
+-- What is the most committed crime in Switzerland?
+SELECT T.offence_criminal_code AS most_common_criminal_offence FROM criminal_offences_registered_by_police AS T JOIN spatial_unit AS S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name='Switzerland'   AND S.country = TRUE   AND T.offence_criminal_code!='Offence - total'   AND T.number_criminal_offences_registered IS NOT NULL GROUP BY T.offence_criminal_code,   T.number_criminal_offences_registered ORDER BY T.number_criminal_offences_registered DESC LIMIT 1;
```

Comment:
Shouldn't it be `ORDER`ed by `SUM`? Since there's no fixed time, I would assume we want the crime with the most occurrences (summed across years). Not sure about this